docfiles/script.html

```diff
@@ -116,6 +116,49 @@ <h2 class="ui item header">@title@</h2>
 									})
 									.modal("show");
 							})
+
+							function tickEvent(id, data) {
+								if (!pxt.aiTrackEvent) return;
+								if (!data) pxt.aiTrackEvent(id);
+								else {
+									var props = {};
+									var measures = {};
+									for (var k in data)
+										if (typeof data[k] == "string") props[k] = data[k];
+										else measures[k] = data[k];
+									pxt.aiTrackEvent(id, props, measures);
+								}
```

Comment:
lol, this is crazy indented:
![image](https://user-images.githubusercontent.com/6453828/69766188-ace24f80-112b-11ea-8ba8-2d8657669526.png)

I don't suppose something could be split into another function?